### PR TITLE
First version of IBM  Federated Learning library

### PR DESCRIPTION
**Pull request by shashank215r**
**Description:** 
    1. This PR contains the first version of IBM Federated learning library for federated learning in enterprise environment.
    2. This PR also includes tutorials for how to set up and use the library.
    3. Working FL examples with instructions are also included.
    4. Github page docs. 

**Supported functionality**

Current version supports following machine learning model types:

1. Neural networks (any neural network topology supported by Keras)
2. Decision Tree ID3
3. Linear classifiers/regressions (with regularizer): logistic regression, linear SVM, ridge regression, Kmeans and Naïve Bayes
4. Deep Reinforcement Learning algorithms including DQN, DDPG, PPO.

Current version supports the following fusion algorithms:

1. Iterative Average          
2. FedAvg [https://arxiv.org/pdf/1602.05629.pdf](url)
3. Gradient Average
4. PFNM [https://arxiv.org/abs/1905.12022](url)
5. Krum [https://papers.nips.cc/paper/6617-machine-learning-with-adversaries-byzantine-tolerant-gradient-descent.pdf](url)
6. Coordinated median [http://www.its.caltech.edu/~whanncar/papers/median.pdf](url)
7. Zeno [https://arxiv.org/abs/1805.10032](url)
8. SPAHM [https://arxiv.org/abs/1911.00218](url)
9. ID3 for Decision Tree

### CODE REVIEW

**Comment by Yi-Zoey** [comment]
> Let's remove all the underscore names files and directories (_data, _pages ...) as well as the index.html from this PR.

@heikoludwig currently `quickstart.md` is under `_pages`, are we going to keep it?

**Comment by heikoludwig** [comment]
Let's just move up QuickStart.md
